packages/blade/src/components/Chip/_decisions/decisions.md

```diff
@@ -0,0 +1,121 @@
+| ----------------------- | -------------------------------------------------------------------- | ------------ | -------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| **children**            | `string`                                                             | `undefined`  | Renders the label text for the Chip                                                                                                                | ✅       |
+| **defaultChecked**      | `boolean`                                                            | `false`      | Sets the default value for an uncontrolled component                                                                                               | ❌       |
+| **icon**                | `Icon`                                                               | `undefined`  | Displays the Blade Icon component within the Chip                                                                                                  | ❌       |
+| **isChecked**           | `boolean`                                                            | `false`      | Controls the Chip's checked state                                                                                                                  | ❌       |
+| **isDisabled**          | `boolean`                                                            | `false`      | Disables or enables the Chip                                                                                                                       | ❌       |
+| **size**                | `'xsmall' \| 'small' \| 'medium' \| 'large'`                         | `'small'`    | Specifies the size of the rendered Chip                                                                                                            | ❌       |
+| **inputType**           | `'checkbox' \| 'radio'`                                              | `'checkbox'` | Determines the Chip's input type                                                                                                                   | ❌       |
+| **value (html native)** | `string`                                                             | `undefined`  | The value of the input field in Chip, [useful in form submissions](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#value) | ❌       |
+| **variant**             | `'positive' \| 'negative' \| 'notice' \| 'information' \| 'neutral'` | `'neutral'`  | Sets the Chip's visual variant                                                                                                                     | ❌       |
```

Comment:
since we are saying Chip cannot be used without ChipGroup, we can also make these props available on ChipGroup so that consumers doesn't have to repeat them each time. 

I think we need to atleast have these props on group: `variant`, `size`, `isDisabled`

```jsx
<ChipGroup variant="positive">
...
</ChipGroup>
```

Similar to how we have in Checkbox and Radio Group

Reply:
@anuraghazra I feel all chips should be of the same size within ChipGroup, hence we can provide `size` prop only for `ChipGroup`. WDYT?

Reply:
Sounds good to me, size/variant probably you can put only on group. But ekbar designers se check kar lena